[BUG] RRLS incompatible

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.35.2

### Operating System

Windows 10

### Minecraft Version

1.21.8

### Describe the bug

Minecraft starts but immediately closes.

### Steps to reproduce

Launch minecraft (fabric 1.21.8)

### Client Log

```shell
---- Minecraft Crash Report ----
// Don't do that.

Time: 2025-12-11 11:17:24
Description: Rendering screen

java.lang.NoClassDefFoundError: Could not initialize class net.ccbluex.liquidbounce.features.module.modules.player.cheststealer.features.FeatureSilentScreen
	at knot//net.minecraft.class_437.handler$dck000$liquidbounce$cancelRenderByChestStealer(class_437.java:5785)
	at knot//net.minecraft.class_437.method_47413(class_437.java)
	at knot//net.minecraft.class_757.mixinextras$bridge$method_47413$95(class_757.java)
	at knot//net.minecraft.class_757.wrapOperation$bjj000$fabric-screen-api-v1$onRenderScreen(class_757.java:3423)
	at knot//net.minecraft.class_757.method_3192(class_757.java:549)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1361)
	at knot//net.minecraft.class_310.method_1514(class_310.java:947)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.ExceptionInInitializerError [in thread "Render thread"]
	at knot//net.ccbluex.liquidbounce.features.module.modules.player.cheststealer.features.FeatureSilentScreen.<init>(FeatureSilentScreen.kt:48)
	at knot//net.ccbluex.liquidbounce.features.module.modules.player.cheststealer.features.FeatureSilentScreen.<clinit>(FeatureSilentScreen.kt)
	at knot//net.minecraft.class_437.handler$dck000$liquidbounce$cancelRenderByChestStealer(class_437.java:5785)
	at knot//net.minecraft.class_437.method_47413(class_437.java)
	at knot//net.minecraft.class_757.mixinextras$bridge$method_47413$95(class_757.java)
	at knot//net.minecraft.class_757.wrapOperation$bjj000$fabric-screen-api-v1$onRenderScreen(class_757.java:3423)
	at knot//net.minecraft.class_757.method_3192(class_757.java:549)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1361)
	at knot//net.minecraft.class_310.method_1514(class_310.java:947)
	... 4 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at knot//net.minecraft.class_437.handler$dck000$liquidbounce$cancelRenderByChestStealer(class_437.java:5785)
	at knot//net.minecraft.class_437.method_47413(class_437.java)
	at knot//net.minecraft.class_757.mixinextras$bridge$method_47413$95(class_757.java)
	at knot//net.minecraft.class_757.wrapOperation$bjj000$fabric-screen-api-v1$onRenderScreen(class_757.java:3423)

-- Screen render details --
Details:
	Screen name: fudge.notenoughcrashes.gui.CrashScreen
	Mouse location: Scaled: (0.000000, 0.000000). Absolute: (0.000000, 0.000000)
	Screen size: ~~ERROR~~ IllegalFormatConversionException: f != java.lang.Integer
Stacktrace:
	at knot//net.minecraft.class_757.method_3192(class_757.java:549)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1361)
	at knot//net.minecraft.class_310.method_1514(class_310.java:947)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:36)
	at knot//net.minecraft.class_310.modify$cbl000$notenoughcrashes$atTheEndOfFirstCatchBeforePrintingCrashReport(class_310.java:14651)
	at knot//net.minecraft.class_310.method_1514(class_310.java:968)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.21.8
	Minecraft Version ID: 1.21.8
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 21.0.7, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 121639208 bytes (116 MiB) / 1509949440 bytes (1440 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 4800H with Radeon Graphics         
	Identifier: AuthenticAMD Family 23 Model 96 Stepping 1
	Microarchitecture: unknown
	Frequency (GHz): 2.89
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce GTX 1650
	Graphics card #0 vendor: NVIDIA
	Graphics card #0 VRAM (MiB): 4096.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 32.0.15.7216
	Memory slot #0 capacity (MiB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 18685.32
	Virtual memory used (MiB): 13080.12
	Swap memory total (MiB): 2432.00
	Swap memory used (MiB): 384.25
	Space in storage for jna.tmpdir (MiB): available: 158772.91, total: 242921.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 158772.91, total: 242921.00
	Space in storage for io.netty.native.workdir (MiB): available: 158772.91, total: 242921.00
	Space in storage for java.io.tmpdir (MiB): available: 158772.91, total: 242921.00
	Space in storage for workdir (MiB): available: 158772.91, total: 242921.00
	JVM Flags: 42 total; -XX:+UnlockExperimentalVMOptions -XX:+UnlockDiagnosticVMOptions -XX:+AlwaysActAsServerClassMachine -XX:+AlwaysPreTouch -XX:+DisableExplicitGC -XX:+UseNUMA -XX:AllocatePrefetchStyle=3 -XX:NmethodSweepActivity=1 -XX:ReservedCodeCacheSize=400M -XX:NonNMethodCodeHeapSize=12M -XX:ProfiledCodeHeapSize=194M -XX:NonProfiledCodeHeapSize=194M -XX:-DontCompileHugeMethods -XX:MaxNodeLimit=240000 -XX:NodeLimitFudgeFactor=8000 -XX:+UseVectorCmov -XX:+PerfDisableSharedMem -XX:+UseFastUnorderedTimeStamps -XX:+UseCriticalJavaThreadPriority -XX:ThreadPriorityPolicy=1 -XX:+UseG1GC -XX:MaxGCPauseMillis=20 -XX:G1HeapRegionSize=16M -XX:G1NewSizePercent=50 -XX:G1MaxNewSizePercent=60 -XX:G1ReservePercent=20 -XX:SurvivorRatio=32 -XX:G1MixedGCCountTarget=3 -XX:G1HeapWastePercent=20 -XX:InitiatingHeapOccupancyPercent=10 -XX:G1RSetUpdatingPauseTimePercent=0 -XX:MaxTenuringThreshold=1 -XX:G1SATBBufferEnqueueingThresholdPercent=30 -XX:G1ConcMarkStepDurationMillis=5.0 -XX:GCTimeRatio=99 -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled -XX:ConcGCThreads=4 -XX:+EagerJVMCI -Xmx4096M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss2M
	Fabric Mods: 
		autoreconnectrf: AutoReconnect 2.0.1+1.21.8
		badoptimizations: BadOptimizations 2.3.1
		cariboustonks: CaribouStonks 0.9.1+1.21.8
			hm-api: HM Api 1.0.1+1.21.2
		chatpatches: Chat Patches 8.0-alpha.5
		clientspoofer: Client Spoofer 1.1.1
		cloth-config: Cloth Config v19 19.0.147
			cloth-basic-math: cloth-basic-math 0.6.1
		continuity: Continuity 3.0.1-beta.1+1.21.6
		dailyrewards: DailyRewards 1.2.2
			com_squareup_okhttp3_okhttp: okhttp 4.12.0
			org_jsoup_jsoup: jsoup 1.17.2
		debugify: Debugify 1.21.8+1.0
		entityculling: EntityCulling 1.9.2
			transition: TRansition 1.0.6
			trender: TRender 1.0.7
		fabric-api: Fabric API 0.136.0+1.21.8
			fabric-api-base: Fabric API Base 0.5.0+cd8c0daa2c
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.101+946bf4c32c
			fabric-biome-api-v1: Fabric Biome API (v1) 16.1.0+c21d71cc2c
			fabric-block-api-v1: Fabric Block API (v1) 1.1.3+946bf4c32c
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.31+946bf4c32c
			fabric-command-api-v2: Fabric Command API (v2) 2.3.1+52fff9ec2c
			fabric-content-registries-v0: Fabric Content Registries (v0) 10.1.0+c75de69d2c
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.41+7f945d5b2c
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.15.6+eb5df52f2c
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.15+946bf4c32c
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.8.14+97214db32c
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 23.2.8+2e1c6d322c
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.20+948fc79a2c
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.1.2+c9e472732c
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 4.0.26+946bf4c32c
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.73+c64c9c5b2c
			fabric-item-api-v1: Fabric Item API (v1) 11.5.0+02ce13f42c
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.2.14+c5f9bbc12c
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.65+946bf4c32c
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.6.4+db4dfd852c
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.56+3f89f5a52c
			fabric-loot-api-v3: Fabric Loot API (v3) 2.0.3+946bf4c32c
			fabric-message-api-v1: Fabric Message API (v1) 6.1.2+946bf4c32c
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 5.2.7+946bf4c32c
			fabric-networking-api-v1: Fabric Networking API (v1) 5.1.0+57cb96dd2c
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 21.1.10+946bf4c32c
			fabric-particles-v1: Fabric Particles (v1) 4.1.10+946bf4c32c
			fabric-recipe-api-v1: Fabric Recipe API (v1) 8.1.17+946bf4c32c
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 6.1.30+946bf4c32c
			fabric-renderer-api-v1: Fabric Renderer API (v1) 7.0.4+208eed172c
			fabric-renderer-indigo: Fabric Renderer - Indigo 4.0.4+946bf4c32c
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.31+fa6cb72b2c
			fabric-rendering-v1: Fabric Rendering (v1) 12.5.1+fbe231d52c
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.25+98e6d6db2c
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.1.12+020423442c
			fabric-screen-api-v1: Fabric Screen API (v1) 2.2.0+e0cb65d02c
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.139+946bf4c32c
			fabric-serialization-api-v1: Fabric Serialization API (v1) 1.0.0+fc0e5aba2c
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.42+946bf4c32c
			fabric-tag-api-v1: Fabric Tag API (v1) 1.2.2+946bf4c32c
			fabric-transfer-api-v1: Fabric Transfer API (v1) 6.0.6+946bf4c32c
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.5.0+c847ce762c
		fabric-language-kotlin: Fabric Language Kotlin 1.13.7+kotlin.2.2.21
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 2.2.21
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 2.2.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 2.2.21
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 2.2.21
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.29.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.10.2
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.10.2
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.7.1
			org_jetbrains_kotlinx_kotlinx-io-bytestring-jvm: kotlinx-io-bytestring-jvm 0.8.0
			org_jetbrains_kotlinx_kotlinx-io-core-jvm: kotlinx-io-core-jvm 0.8.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.9.0
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.9.0
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.9.0
		fabricloader: Fabric Loader 0.18.1
			mixinextras: MixinExtras 0.5.0
		ferritecore: FerriteCore 8.0.0
		fnp_patcher: FNP Patcher 25.9.2+1.21.5
			one_pkg_sewlia-config: sewlia-config 1.0.0-20250810.044244-13
			one_pkg_velocity_rc_velocity-native: velocity-native 3.4.0-SNAPSHOT
			org_yaml_snakeyaml: snakeyaml 2.4
		forgeconfigapiport: Forge Config API Port 21.8.1
			com_electronwill_night-config_core: core 3.8.3
			com_electronwill_night-config_toml: toml 3.8.3
		fzzy_config: Fzzy Config 0.7.3+1.21.6
			blue_endless_jankson: jankson 1.2.3
			fabric-permissions-api-v0: fabric-permissions-api 0.4.0
			net_peanuuutz_tomlkt_tomlkt-jvm: tomlkt-jvm 0.3.7
		hypixel-mod-api: Hypixel Mod API 1.0.1+build.1+mc1.21
			net_hypixel_mod-api: mod-api 1.0.1
		hypixelautotip: Hypixel AutoTip 1.1.3
		hypixelcry: Hypixel Cry 1.1.3
			com_logisticscraft_occlusionculling: occlusionculling 0.0.8-SNAPSHOT
			io_github_spair_imgui-java-binding: imgui-java-binding 1.84.1.1
			io_github_spair_imgui-java-lwjgl3: imgui-java-lwjgl3 1.84.1.1
			io_github_spair_imgui-java-natives-linux: imgui-java-natives-linux 1.84.1.1
			io_github_spair_imgui-java-natives-macos: imgui-java-natives-macos 1.84.1.1
			io_github_spair_imgui-java-natives-windows: imgui-java-natives-windows 1.84.1.1
			moe_nea_neurepoparser: neurepoparser 1.8.0
			net_azureaaron_legacy-item-dfu: legacy-item-dfu 1.0.3+1.21.5
			org_eclipse_jgit_org_eclipse_jgit: org.eclipse.jgit 7.3.0.202506031305-r
			org_luaj_luaj-jse: luaj-jse 3.0.1
		java: OpenJDK 64-Bit Server VM 21
		krypton: Krypton 0.2.9
			com_velocitypowered_velocity-native: velocity-native 3.4.0-SNAPSHOT
		liquidbounce: LiquidBounce 0.35.2
			ai_djl_api: api 0.34.0
			ai_djl_pytorch_pytorch-engine: pytorch-engine 0.34.0
			com_github_ccbluex_discordipc: DiscordIPC 4.0.0
			com_kohlschutter_junixsocket_junixsocket-common: junixsocket-common 2.8.3
			com_kohlschutter_junixsocket_junixsocket-native-common: junixsocket-native-common 2.8.3
			com_squareup_okhttp3_okhttp-coroutines: okhttp-coroutines 5.1.0
			com_squareup_okhttp3_okhttp-jvm: okhttp-jvm 5.1.0
			com_squareup_okio_okio-jvm: okio-jvm 3.15.0
			com_thealtening_api_api: api 4.1.0
			com_vdurmont_semver4j: semver4j 3.1.0
			io_netty_netty-codec-http: netty-codec-http 4.1.118
			io_netty_netty-codec-socks: netty-codec-socks 4.1.118
			io_netty_netty-handler-proxy: netty-handler-proxy 4.1.118
			io_netty_netty-transport-classes-kqueue: netty-transport-classes-kqueue 4.1.118
			io_netty_netty-transport-native-kqueue_osx-aarch_64: netty-transport-native-kqueue 4.1.118
			io_netty_netty-transport-native-kqueue_osx-x86_64: netty-transport-native-kqueue 4.1.118
			mcef: MCEF (Minecraft Chromium Embedded Framework) 3.1.6-1.21.10
			net_ccbluex_fastutil4k-more-collections: fastutil4k-more-collections 0.2.2
			net_ccbluex_mc-authlib: mc-authlib 1.5.2
			net_ccbluex_netty-httpserver: netty-httpserver 2.4.2
			net_fabricmc_tiny-mappings-parser: tiny-mappings-parser 0.3.0+build.17
			net_lenni0451_reflect: Reflect 1.3.2
			org_ahocorasick_ahocorasick: ahocorasick 0.6.3
			org_apache_tika_tika-core: tika-core 2.9.2
			org_graalvm_js_js-language: js-language 25.0.0
			org_graalvm_polyglot_polyglot: polyglot 25.0.0
			org_graalvm_regex_regex: regex 25.0.0
			org_graalvm_sdk_collections: collections 25.0.0
			org_graalvm_sdk_jniutils: jniutils 25.0.0
			org_graalvm_sdk_nativeimage: nativeimage 25.0.0
			org_graalvm_sdk_word: word 25.0.0
			org_graalvm_shadowed_icu4j: icu4j 25.0.0
			org_graalvm_shadowed_json: json 25.0.0
			org_graalvm_shadowed_xz: xz 25.0.0
			org_graalvm_tools_chromeinspector-tool: chromeinspector-tool 25.0.0
			org_graalvm_tools_coverage-tool: coverage-tool 25.0.0
			org_graalvm_tools_dap-tool: dap-tool 25.0.0
			org_graalvm_tools_insight-heap-tool: insight-heap-tool 25.0.0
			org_graalvm_tools_insight-tool: insight-tool 25.0.0
			org_graalvm_tools_lsp-tool: lsp-tool 25.0.0
			org_graalvm_tools_lsp_api: lsp_api 25.0.0
			org_graalvm_tools_profiler-tool: profiler-tool 25.0.0
			org_graalvm_truffle_truffle-api: truffle-api 25.0.0
			org_graalvm_truffle_truffle-compiler: truffle-compiler 25.0.0
			org_graalvm_truffle_truffle-runtime: truffle-runtime 25.0.0
		minecraft: Minecraft 1.21.8
		modelfix: Model Gap Fix 1.21.5-1.12
		modernfix: ModernFix 5.20.4-beta.54+mc1.21.8.c05a
		modmenu: Mod Menu 15.0.0
			placeholder-api: Placeholder API 2.7.2+1.21.8
		no-resource-pack-warnings: No Resource Pack Warnings 1.3.1
		nofrills: NoFrills 0.4.8
			de_topobyte_jama: jama 1.1.0
			meteordevelopment_orbit: orbit 0.2.4
			owo: oωo 0.12.22+1.21.8
				io_wispforest_endec: endec 0.1.9
				io_wispforest_endec_gson: gson 0.1.6
				io_wispforest_endec_jankson: jankson 0.1.6
				io_wispforest_endec_netty: netty 0.1.5
		notenoughcrashes: Not Enough Crashes 4.4.9+1.21.8
		packetfixer: Packet Fixer 3.3.0
		particle_core: Particle Core 0.2.6+1.21.6
			conditional-mixin: conditional mixin 0.5.1
		rrls: Remove Reloading Screen 5.1.10+mc1.21.8-fabric
		simpleblockoverlay: Simple Block Overlay 1.4.8
		simplenickhider: Simple Nick Hider 0.1.2
		skyblocker: Skyblocker 5.9.0-beta.1+1.21.8
			dandelion: Dandelion 1.0.0-alpha.9+1.21.8
			meteordevelopment_discord-ipc: discord-ipc 1.1
			net_azureaaron_networth-calculator: networth-calculator 1.0.5
			org_apache_commons_commons-math3: commons-math3 3.6.1
			org_apache_commons_commons-text: commons-text 1.13.1
		skyhanni: SkyHanni 5.4.0
			libninepatch: LibNinePatch 1.2.0
		toadlib: ToadLib 1.3.6
		yet_another_config_lib_v3: YetAnotherConfigLib 3.7.1+1.21.6-fabric
			com_twelvemonkeys_common_common-image: common-image 3.12.0
			com_twelvemonkeys_common_common-io: common-io 3.12.0
			com_twelvemonkeys_common_common-lang: common-lang 3.12.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.12.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.12.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.12.0
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
	Client Crashes Since Restart: 1
	Integrated Server Crashes Since Restart: 0
	Suspected Mods: Minecraft (minecraft), Fabric Loader (fabricloader), LiquidBounce (liquidbounce), Fabric Screen API (v1) (fabric-screen-api-v1), Not Enough Crashes (notenoughcrashes)
```

### Screenshots

<img width="557" height="757" alt="Image" src="https://github.com/user-attachments/assets/63615c03-14b8-4894-89b4-f9ce15103636" />

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -125,6 +125,8 @@ dependencies {
     // Recommended mods (on IDE)
     modApi(libs.sodium)
     modApi(libs.lithium)
+    modRuntimeOnly(libs.immediatelyFast)
+    modRuntimeOnly(libs.iris)
 
     // ViaFabricPlus
     modApi(libs.vfp.api)

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -36,6 +36,9 @@ exploit_preventer_mod = "0.1.7"
 sodium = "mc1.21.11-0.8.1-fabric"
 lithium = "mc1.21.11-0.21.1-fabric"
 viafabricplus = "4.4.1"
+# Mods for compatibility test (runtime only)
+immediatelyFast = "1.14.1+1.21.11-fabric"
+iris = "1.10.3+1.21.11-fabric"
 
 # Kotlin
 # https://kotlinlang.org/releases.html
@@ -92,6 +95,8 @@ vfp-api = { group = "com.viaversion", name = "viafabricplus-api", version.ref = 
 vfp = { group = "com.viaversion", name = "viafabricplus", version.ref = "viafabricplus" }
 exploitPreventer-api = { group = "com.nikoverflow", name = "ExploitPreventer-API", version.ref = "exploit_preventer_api" }
 exploitPreventer = { group = "maven.modrinth", name = "exploitpreventer", version.ref = "exploit_preventer_mod" }
+immediatelyFast = { group = "maven.modrinth", name = "ImmediatelyFast", version.ref = "immediatelyFast" }
+iris = { group = "maven.modrinth", name = "iris", version.ref = "iris" }
 
 [plugins]
 fabric-loom = { id = "fabric-loom", version.ref = "fabric-loom" }

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/client/MixinMinecraft.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/client/MixinMinecraft.java
@@ -301,7 +301,8 @@ public abstract class MixinMinecraft {
     @Redirect(method = "setScreen", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/MouseHandler;releaseMouse()V"))
     private void cancelScreenMouseForChestStealer(MouseHandler instance) {
         // Allows rotation.
-        if (!LiquidBounce.INSTANCE.isInitialized() || !FeatureSilentScreen.getShouldHide() || FeatureSilentScreen.getUnlockCursor()) {
+        if (!LiquidBounce.INSTANCE.isInitialized() ||
+            !FeatureSilentScreen.INSTANCE.getShouldHide() || FeatureSilentScreen.INSTANCE.getUnlockCursor()) {
             instance.releaseMouse();
         }
     }

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinAbstractContainerScreen.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinAbstractContainerScreen.java
@@ -83,7 +83,7 @@ public abstract class MixinAbstractContainerScreen<T extends AbstractContainerMe
             ci.cancel();
         }
 
-        if (FeatureSilentScreen.getShouldHide()) {
+        if (FeatureSilentScreen.INSTANCE.getShouldHide()) {
             ci.cancel();
         }
     }

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinScreen.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinScreen.java
@@ -94,7 +94,7 @@ public abstract class MixinScreen implements ScreenAddition {
 
     @Inject(method = "renderWithTooltipAndSubtitles", at = @At("HEAD"), cancellable = true)
     private void cancelRenderByChestStealer(CallbackInfo ci) {
-        if (FeatureSilentScreen.getShouldHide()) {
+        if (FeatureSilentScreen.INSTANCE.getShouldHide()) {
             ci.cancel();
         }
     }

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinScreen.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinScreen.java
@@ -45,8 +45,6 @@ import javax.annotation.Nullable;
 
 @Mixin(Screen.class)
 public abstract class MixinScreen implements ScreenAddition {
-    @Shadow
-    protected abstract void removeWidget(GuiEventListener child);
 
     @Final
     @Shadow
@@ -94,7 +92,7 @@ public abstract class MixinScreen implements ScreenAddition {
 
     @Inject(method = "renderWithTooltipAndSubtitles", at = @At("HEAD"), cancellable = true)
     private void cancelRenderByChestStealer(CallbackInfo ci) {
-        if (FeatureSilentScreen.INSTANCE.getShouldHide()) {
+        if (LiquidBounce.INSTANCE.isInitialized() && FeatureSilentScreen.INSTANCE.getShouldHide()) {
             ci.cancel();
         }
     }

--- a/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/cheststealer/features/FeatureSilentScreen.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/cheststealer/features/FeatureSilentScreen.kt
@@ -47,7 +47,6 @@ import net.minecraft.world.phys.Vec3
  */
 object FeatureSilentScreen : ToggleableConfigurable(ModuleChestStealer, "SilentScreen", false) {
 
-    @get:JvmStatic
     val unlockCursor by boolean("UnlockCursor", false)
 
     private val drawInventoryTag = object : ToggleableConfigurable(this, "DrawInventoryTag", enabled = true) {
@@ -101,17 +100,19 @@ object FeatureSilentScreen : ToggleableConfigurable(ModuleChestStealer, "SilentS
         tree(drawInventoryTag)
     }
 
-    @get:JvmStatic
     var shouldHide = false
+        private set
 
-    val screenHandler = handler<ScreenEvent> { event ->
+    @Suppress("unused")
+    private val screenHandler = handler<ScreenEvent> { event ->
         shouldHide = event.screen?.canBeStolen() == true
     }
 
     @Volatile
     private var lastInteractedBlock: BlockPos? = null
 
-    val packetHandler = handler<PacketEvent> { event ->
+    @Suppress("unused")
+    private val packetHandler = handler<PacketEvent> { event ->
         val packet = event.packet
         // TODO: handle other interactions
         if (packet is ServerboundUseItemOnPacket && packet.hitResult.type === HitResult.Type.BLOCK) {

--- a/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/cheststealer/features/FeatureSilentScreen.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/cheststealer/features/FeatureSilentScreen.kt
@@ -42,8 +42,8 @@ import net.minecraft.core.BlockPos
 import net.minecraft.world.phys.Vec3
 
 /**
- * @see net.ccbluex.liquidbounce.injection.mixins.minecraft.client.MixinMinecraftClient
- * @see net.ccbluex.liquidbounce.injection.mixins.minecraft.gui.MixinHandledScreen
+ * @see net.ccbluex.liquidbounce.injection.mixins.minecraft.client.MixinMinecraft
+ * @see net.ccbluex.liquidbounce.injection.mixins.minecraft.gui.MixinAbstractContainerScreen
  */
 object FeatureSilentScreen : ToggleableConfigurable(ModuleChestStealer, "SilentScreen", false) {
 

